docs/standard/serialization/snippets/system-text-json-how-to/vb/WeatherForecastRequiredPropertyConverter.vb

```diff
@@ -0,0 +1 @@
+﻿' https://docs.microsoft.com/dotnet/standard/serialization/system-text-json-how-to#visual-basic-support
```

Comment:
What does this single line mean? Is this vb file rendered in any way?

Reply:
It renders as a VB snippet consisting only of this comment for snippets that can't be done in VB. In case the person looking for VB missed the explanation of what VB can't be used for, it links to the VB note in the base how-to doc. I can add text like 'This code example can't be done in Visual Basic', though that won't get localized.